testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -378,6 +378,8 @@
 
 Given(/^I am not authorized$/) do
   begin
+    xpath_logout = '//a[@href=\'/rhn/Logout.do\']'
```

Comment:
Why do we have this change ?

Reply:
Using a different user, but keeping the same session it gonna require that we logout before login.